packages/cli/webpack/src/config/client.ts

```diff
@@ -280,6 +281,16 @@ class ClientWebpackConfig extends BaseWebpackConfig {
         },
       ]);
     }
+
+    if (this.options.cliOptions?.analyze) {
```

Comment:
It may only be better in the production environment

Reply:
--analyze  is only related to build command,  not the execution environment.